### PR TITLE
nolo tlc ciel 13_3_1

### PR DESCRIPTION
**Pull request by nolo16**
happy halloween, shiki!

### CODE REVIEW

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 1]
```diff
@@ -1,17 +1,17 @@
 [sha:49abf0681645852608461c13fd0e2b27707c3b6e]{
 -- Page 10029, Offset 33898.
 -- 屈み込んだ姿勢から、全力で、前に向かって走り出した。
-From a leaned-in stance, I rush forward with all my strength.
+From my crouching position, I start rushing forward as fast as I can.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:4]: I start to rush forward as fast as I can.

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 }
 [sha:8c04894277740d8a2a29aa604f5d3901f783aab6]{
 -- Page 10029, Offset 33899.
 -- 後ろを向けば―――
-After all, I have a feeling that if I try to turn back―――
+After all, I have a feeling that if I try to retreat backwards―――
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:9]: revert

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 }
 [sha:4dc4dbf4fb01eb0bc45f6d8576a02cd2b90f60e4]{
 -- Page 10029, Offset 33900.
 -- 背中を見せれば無防備な背後から、心臓を撃たれる事を感じ取っていたからだ。
-She would shoot through my heart the moment she sees my defenseless back.
+she'll pierce through my heart the moment I show her my defenseless back.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:14]: drop "through"

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 4]
```diff
@@ -26,27 +26,27 @@ The gun barrel passes right overhead.
...
 -- Page 251, Offset 33903.
 -- 低い姿勢と、近すぎる距離が幸いした。
-I was fortunate to be at a low angle and at a distance far too close to her.
+Fortunately, the fact that I'm stooped forward and within her close proximity is working to my advantage here.
 }
 [sha:fac7816af3e9c37598c3ce447c40297bb25769e0]{
 -- Page 252, Offset 33904.
 -- 俺は顔面に突き出された銃口を紙一重でかいくぐり、地面を踏む足裏の重心を横にズラした。
-I slip past the gun point aimed at my face by a hair's breadth, and shift the center of gravity of my foot stepping on the ground to the side.
+I narrowly avoid the muzzle that's aimed at my head by a hair's breadth, and then I shift the center of gravity of my foot horizontally.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:34]: horizontally shift my foot's center of gravity.

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 5]
```diff
@@ -26,27 +26,27 @@ The gun barrel passes right overhead.
...
 }
 [sha:b8f9315269e1cd0ee4f724ad2c4762876b89e8c2]{
 -- Page 252, Offset 33905.
 -- このまま突進すれば正面からぶつかるだけだ。タックルのカタチで押し倒す事も考えたが却下する。
-If I charge in at this rate, I'll simply collide with her from the front. I thought about pushing her down with a tackle, but I decided against it.
+If I keep charging in, it'll only result in a head-on collision. I considered pushing her down with a tackle, but I decided against it.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:39]: but decided against it

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 6]
```diff
@@ -26,27 +26,27 @@ The gun barrel passes right overhead.
...
 -- Page 253, Offset 33906.
 -- あれだけの重機を片手で持つ相手だ。組み合っての力勝負は選択肢に含まれない。
-She's an opponent who can carry that heavy machinery with a single hand. Going up against her in a grappling match isn't an option.
+She's capable of wielding that enormous weapon with a single hand. I won't get anywhere in a battle of brute strength.
 }
 [sha:f4b10fc28b5a3732c8633c9b4308c4d097e43885]{
 -- Page 253, Offset 33907.
 -- 俺は体をわずかに横に逸らして、彼女の真横を―――
-I'll slightly turn my body to her side and―――
+I'll twist my body to the side slightly, and then slip through her flank―――
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:49]: drop then

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 7]
```diff
@@ -56,27 +56,27 @@ I'll slightly turn my body to her side and―――
 [sha:519e7ac0cf50bb26ebf7150238d0ff24f1c3f6ef]{
 -- Page 254, Offset 33909.
 -- 真横をすり抜けようとした矢先、先輩の動きは一変した。
-Right as I was about to slip past, Senpai changes her course of action completely.
+Just as I'm about to slip past her, Senpai changes her course of action completely.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:59]: is that right for "先輩の動きは一変した"? I think he's saying she changed her physical movements, not her approach judging by the following lines.
> completely changes her movements.

pretty clunky but imo closer, right?

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 8]
```diff
@@ -111,12 +111,12 @@ It's too much for me to understand.
 [sha:bf9b744984f4f72735589af2dc6f1d540ed2c622]{
 -- Page 258, Offset 33920.
 -- 左肘から先が山形にひん曲がっている。
-My left arm from the elbow onwards twists into the shape of a mountain.
+From the elbow onward, most of my left arm has been bent into the shape of a cone.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:114]: whats the cg here? "bent into the shape of a cone" doesnt really make any sense. I get the feeling what he's saying is either that:
- his arm has been all twisted up and his hand has been ripped off exposing bone (reading from 曲がっている)
- his arm snapped off, again exposing bone.

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 9]
```diff
@@ -181,7 +181,7 @@ The woman looks at me with uninterested eyes.
 [sha:aa0acad8179f763ef766e4b8c9ef25fda76127de]{
 -- Page 263, Offset 33934.
 -- 先輩は、俺をいつでも殺す事ができると、見下している。
-Senpai looks down on me as if to say she could kill me at any moment.
+Senpai is looking down on me, as if to say that she could kill me at any moment if she so pleased.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:184]: where is "if she so pleased" coming from? "いつでも殺す事ができる" is more like "she can kill me at any time"

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 10]
```diff
@@ -411,17 +411,17 @@ But I'm still alive.
...
 -- Page 273, Offset 33980.
 -- なんとか生きてる。
-I'm still alive somehow.
+Somehow, I'm still alive.
 }
 [sha:ded838f6b32fe18f6ca0b020f3bc5f3b5f5198b7]{
 -- Page 274, Offset 33981.
 -- 体。体は、また先輩から離れている。
-My body. My body is separated from Senpai once again.
+My body. Once again, my body is separated from Senpai's.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:419]: this structure doesn't really parse in english. Maybe something like
> My body isーーーonce again, separated from Senpai's.

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 11]
```diff
@@ -431,52 +431,53 @@ It smells like gunpowder coming out of my shoulder.
...
 -- Page 274, Offset 33984.
 -- あの銃口が突き刺さった瞬間、引き金を引かれたのか。
-She must have pulled the trigger as soon as she thrust that gun point into me.
+She must have pulled the trigger as soon as she thrust the muzzle into me.
 }
 [sha:896c7526252a7b6c378922cd7989f152e8560976]{
 -- Page 274, Offset 33985.
 -- それで俺は吹っ飛んで、また先輩と距離が離れている。
-Because of that, I was blown away, creating a distance between us once again.
+I was blown away, and now we're separated by a distance again.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:439]: and now we're once again separated by some distance.

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 12]
```diff
@@ -431,52 +431,53 @@ It smells like gunpowder coming out of my shoulder.
...
 [sha:8f2e2ca02d3f84d91d9de8b07809a73feff43b11]{
 -- Page 277, Offset 33991.
 -- 「紙……紙で出来てる、銃弾、とか」
-"Paper... you're telling me, that bullet, was made of paper?"
+"Paper... you're telling me... that bullet... was made of paper?"
 }
 [sha:ae9e6b4a977d1cb01c55540d65476527d5940076]{
 -- Page 277, Offset 33992.
 -- デタラメにも程がある。
+// let shiki say bullshit!!!
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:474]: i'm all for it. shiki can say bullshit for デタラメ

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 13]
```diff
@@ -566,12 +567,12 @@ There's no doubt that this next one――――
 [sha:d695fbd4cb3355e06dda7fcb3f6b8780709275f9]{
 -- Page 282, Offset 34011.
 -- あの銃口が顔の中心を撃ち抜き、鉄の釘が鼻っ柱から後頭部に抜ける絵を想像した。
-I can imagine the gun point shooting at the center of my head, and the iron nail passing through my nose and out of the back of my head.
+I can already imagine being shot right at the center of my head, with the iron nail piercing straight through my nose bridge and exiting through the back of my head.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:570]: the bridge of my nose

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 14]
```diff
@@ -836,17 +837,17 @@ Senpai speaks to me from behind.
...
 -- Page 301, Offset 34065.
 -- 「まだ解らないんですか。貴方は今どのくらいの速さで走ったと思っているんです。致命傷を受けて、どうして死なないでいるんです。
-"You still don't get it, do you? How fast did you think you ran just now? Why haven't you died after taking a fatal wound?
+"You still don't get it, do you? Do you not know how fast you were running just now? Do you not know why you haven't died even after sustaining injuries that should've been fatal?
 }
 [sha:b43aebe5775a4c80dd7ad461134a66d4980dc23c]{
 -- Page 301, Offset 34066.
 -- ―――貴方は箱の中のマウスと変わらない。
-―――You're no different than a mouse trapped inside a box.
+―――You're no different from a mouse trapped inside a box.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:845]: trapped in a box

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 15]
```diff
@@ -836,17 +837,17 @@ Senpai speaks to me from behind.
...
 }
 [sha:58eb57a107e5e2528d75b5c493c94ba5383cdac1]{
 -- Page 301, Offset 34067.
 -- なのに、まだ、何から逃げ出そうというんですか？」
-But despite that, you still intend on running away?"
+But despite that, do you still intend on continuing to try to run away?"
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:850]: drop "do"

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 16]
```diff
@@ -856,17 +857,17 @@ But despite that, you still intend on running away?"
...
 -- Page 302, Offset 34070.
 -- 胸を貫通した傷口は止血されている。
-The wound from my pierced chest has stopped bleeding.
+It pierced straight through my chest, but the wound has already stopped bleeding.
 }
 [sha:596f804abe87c9b9ad267c5dd7d16c8524c6c9d5]{
 -- Page 302, Offset 34071.
 -- 俺はもう、完全に人間ではなくなっている―――
-I've already turned into something that's not completely human―――
+I've already lost any semblance of humanity that I once had―――
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:870]: the humanity I once had

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt [hunk 17]
```diff
@@ -951,17 +952,17 @@ The executioner inches closer, showing off the sounds of her footsteps.
...
 -- Page 307, Offset 34088.
 -- 「いまさら格好つけなくていいですよ。
-"You don't have to show off anymore.
+"You can stop putting on airs now.
 }
 [sha:9382f7fd7674321d1b718ad844f5326110228a5e]{
 -- Page 307, Offset 34089.
 -- 貴方は無邪気に目の前のエサに飛びついて自滅するんですから。ほら。学校では、いつだってそうしていたでしょう？」
-After all, you ruined yourself by carelessly jumping at the bait in front of you. Think about it. You've been doing it all the time at school, right?"
+After all, you sealed your own fate by carelessly jumping at the bait that was dangled right before your eyes. Think about it. Haven't you been doing the same thing all this time at school?"
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:960]: 無邪気に目の前のエサに飛びついて is more like
> innocently taking the bait

and it should be "that I dangled right before your eyes" or "that was dangling right before your eyes"

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt (hunk 8 above)
Comment (nolo16) [reply script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:114]: image
i genuinely have no idea

what do you think?

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt (hunk 12 above)
Comment (nolo16) [reply script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:474]: consider it done

File: script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt (hunk 8 above)
Comment (Avuxo) [reply script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt:114]: I think leave a comment for the boys in dover and move on

**Pull request merged**